memecity.engine.ui/UI/Menu/Menu.h

```diff
@@ -4,11 +4,16 @@
 #include <memory>
 #include <Engine.h>
 #include "MenuItem.h"
+#include "../../../memecity.game/Assets.h"
 
 namespace memecity::engine::ui::menu {
 	class Menu
 	{
 	private:
+		static const int title_height = 50;
+		static const int menu_item_height = 30;
+
+		std::vector<std::unique_ptr<MenuItem>> read_only_menu_items;
```

Comment:
Ik zou het mooier vinden als de `MenuItem` zelf beslist of het read only is of niet.